### PR TITLE
Add travis config

### PR DESCRIPTION
**Pull request by silvolu**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (smishra2) [inline .travis.yml]: Is it convention or necessary to leave an empty first line?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+language: node_js
+node_js:
+  - "lts/*"
+install:
+    - npm install
+script:
+    - npm run lint
```
Comment (smishra2) [inline .travis.yml]: Do we need this if linting is part of test script?

Comment (smishra2) [reply .travis.yml]: Additionally, if only 'test' is used, it [seems](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Default-Build-Script) like that's the default, so maybe can be omitted

**Comment by silvolu** [comment]
@smishra2 PTAL

**Pull request merged**